docs/known_issues/index.md

```diff
@@ -15,18 +15,17 @@ The following compiler issues are known and we are working on it. Please do not
 
 ### Release
 
-resolc `0.1.0-dev.7`
+resolc `0.1.0-dev.10`
 
 ### Missing features
 
 - [Libraries with public functions are not supported](https://github.com/paritytech/revive/issues/91)
 - [Automatic import resolution is not supported](https://github.com/paritytech/revive/issues/98)
 - The emulated EVM linear contract memory is limited to 64kb in size. Will be fixed with support for metered dynamic memory.
-- [Gas limits for contract calls are ignored](https://github.com/paritytech/revive/issues/117)
```

Comment:
Can you add an explanation of the cross contract call behaviour somewhere in the "differences to eth chapter"? Like that they are uncapped and we have this heuristic. And will be adding a pre-compile to limit them later.

Reply:
The "differences to eth chapter" is really lacking. Because I basically didn't document anything there yet. And it's starting to surface as we get closer to a production development. So I plan to write a lot more there in the coming days. Do you still want me to include this in this PR?

Reply:
> So I plan to write a lot more there in the coming days.

Great. I imagine the information regarding contract instantiation would also be on your list then. Then do it as a follow up.

Reply:
Yes that's also on my list.

Reply:
I'll do this in the follow up then, thanks!